runtime/test/planner-tests.js

```diff
@@ -654,4 +657,81 @@ describe('Automatic handle resolution', function() {
     assert.equal('use', handle.fate);
     assert.equal('test:1', handle.id);
   });
+  it('composes recipe rendering a list of items from a recipe', async () => {
+    let arc = null;
+    let recipes = await verifyResolvedPlans(`
+      import 'artifacts/Common/List.manifest'
+      schema Thing
+
+      particle ThingProducer
+        out [Thing] things
+
+      particle ThingRenderer
+        in Thing thing
+        consume item
+
+      recipe ProducingRecipe
+        create as things
+        ThingProducer`, arcRef => arc = arcRef);
+
+    assert.lengthOf(recipes, 2);
+    const composedRecipes = recipes.filter(r => r.name !== 'ProducingRecipe');
+    assert.lengthOf(composedRecipes, 1);
+
+    assert.equal(composedRecipes[0].toString(), `recipe
+  create as handle0 // [Thing {}]
+  create #selected as handle1 // Thing {}
+  copy '${arc.id}:particle-literal:ThingRenderer' as handle2 // HostedParticleShape
+  slot 'r0' #root as slot1
+  ItemMultiplexer as particle0
+    hostedParticle = handle2
+    list <- handle0
+    consume item as slot0
+  SelectableList as particle1
+    items = handle0
+    selected = handle1
+    consume root as slot1
+      provide action as slot2
+      provide annotation as slot3
+      provide item as slot0
+      provide postamble as slot4
+      provide preamble as slot5
+  ThingProducer as particle2
+    things -> handle0`);
+  });
+  it.only('composes recipe rendering a list of items from the current arc', async () => {
```

Comment:
Remove `only`?

Reply:
yeah, other tests might be useful as well, removed.